Octokit.Tests.Conventions/ModelTests.cs

```diff
@@ -107,6 +107,50 @@ public void ResponseModelsHaveReadOnlyCollections(Type modelType)
             }
         }
 
+        [Theory]
+        [MemberData("ResponseModelTypes")]
+        public void ResponseModelsHaveUrlPropertiesOfTypeString(Type modelType)
+        {
+            var propertiesWithInvalidType = modelType
+                .GetProperties()
+                .Where(x => x.Name.EndsWith("Url"))
+                .Where(x => x.PropertyType != typeof(string))
+                .ToList();
+
+            if (propertiesWithInvalidType.Count > 0)
+            {
+                throw new InvalidUrlPropertyTypeException(modelType, typeof(string), propertiesWithInvalidType);
+            }
+        }
+
+        [Theory]
+        [MemberData("RequestModelTypes")]
+        public void RequestModelsHaveUrlPropertiesOfTypeUri(Type modelType)
+        {
+            if (modelType == typeof(AuthorizationUpdate) ||
+                modelType == typeof(NewAuthorization) ||
+                modelType == typeof(NewCommitStatus) ||
```

Comment:
No worries at all, thanks for spotting this. I didn't even know C# 6 was explicitly disabled. I'll take a look at the places in the [`dotnetcore`](https://github.com/octokit/octokit.net/tree/dotnetcore) branch tomorrow and use new features where possible.